src/evo/specialtxman.cpp

```diff
@@ -50,9 +50,6 @@ static bool CheckSpecialTxInner(CDeterministicMNManager& dmnman, const Chainstat
         case TRANSACTION_MNHF_SIGNAL:
             return CheckMNHFTx(chainman, qman, tx, pindexPrev, state);
         case TRANSACTION_ASSET_LOCK:
-            if (!DeploymentActiveAfter(pindexPrev, consensusParams, Consensus::DEPLOYMENT_V20)) {
-                return state.Invalid(TxValidationResult::TX_CONSENSUS, "assetlocks-before-v20");
-            }
```

Comment:
4afc8539111d30d702b7bfbc0a2ff1ac306715cb: I mean, this is kinda a breaking change no? Wouldn't affect mainnet but...

Reply:
https://github.com/dashpay/dash/commit/4afc8539111d30d702b7bfbc0a2ff1ac306715cb it would not affect mainnet, testnet, indeed.